src/sentry/tagstore/base.py

```diff
@@ -79,6 +79,8 @@ class TagStorage(Service):
 
         'get_tag_value_qs',
         'get_group_tag_value_qs',
+
+        'get_groups_times_seen_by_environment_name',
```

Comment:
Not sure if this is more appropriate to associate with one of the other groups above.